servers/src/main/java/tachyon/Format.java

```diff
@@ -74,14 +74,15 @@ public static void main(String[] args) throws IOException {
           Constants.RAW_TABLE_MASTER_NAME,
       };
       for (String masterServiceName : masterServiceNames) {
-        if (!formatFolder(masterServiceName + "JOURNAL_FOLDER", PathUtils.concatPath(masterJournal,
+        if (!formatFolder(masterServiceName + "_JOURNAL_FOLDER", PathUtils.concatPath(masterJournal,
             masterServiceName), tachyonConf)) {
           System.exit(-1);
         }
       }
 
       UnderFileSystemUtils.touch(
-          masterJournal + Constants.FORMAT_FILE_PREFIX + System.currentTimeMillis(), tachyonConf);
+          PathUtils.concatPath(masterJournal, Constants.FORMAT_FILE_PREFIX
```

Comment:
let's say, `masterJournal` is `/test`, in original code, the format file is created as `/testXXXX` instead of `/test/XXXX`, which causes Tachyon think the folder `/test` is not formatted because format file `/test/XXXX` can not be found.